src/main/java/dev/unnm3d/rediseconomy/config/Settings.java

```diff
@@ -23,6 +23,9 @@ public class Settings {
     @Comment({"if true, migrates the bukkit offline uuids accounts to the default RedisEconomy currency",
             "During the migration, the plugin will be disabled. Restart all RedisEconomy instances after the migration."})
     public boolean migrationEnabled = false;
+    @Comment({"if enabled, the plugin will migrate economy data from sql database",
+            "During the migration, the plugin will be disabled. Restart all RedisEconomy instances after the migration."})
+    public SqlMigrateSettings sqlMigration = new SqlMigrateSettings(false, "com.mysql.cj.jdbc.Driver", "jdbc:mysql://localhost:3306/database?useSSL=false", "root", "password", "economy", "name", "uuid", "money");
```

Comment:
Merge sqlMigration to the same data structure as migrationEnabled
Make the record so that we'll have only a "enabled" key and a "type" key (SQL type or OfflinePlayer type)